draft-ralston-mimi-terminology.md

```diff
@@ -75,86 +75,84 @@ In some cases, the user identity can be synonymous with the user ID, however the
-  these types of chats are discovered through a directory or website. Chats which require
-  a request to join, or knock, are considered public chats. Sometimes these will be referred
+* **Public Room**: A group chat which does not require an invite to participate. Usually
+  these types of rooms are discovered through a directory or website. Rooms which require
+  a request to join, or knock, are considered public rooms. Sometimes these will be referred
   to as **Public Chatrooms**.
 
-* **Auditorium Chat**: Either a group chat or public chat where most users are unable to
-  send events and cannot be seen as chat members by other users. When an event is sent,
-  it may appear to be sent by the chat itself rather than the specific user who sent it.
-  Sometimes these are called **Auditorium Rooms**.
+* **Auditorium Room**: Either a group chat or public room where most users are unable to
+  send events and cannot be seen as room participants by other users. When an event is sent,
+  it may appear to be sent by the room itself rather than the specific user who sent it.
+  Sometimes these are called **Auditorium Chats**.
 
-Depending on the system, a chat's type can be mutable. For example, a user may be permitted
-to introduce new users to a group DM to implicitly convert it to a group chat, or they
-simply may be unable to implicitly or explicity change the chat type.
+Depending on the system, a room's type can be mutable. For example, a user may be permitted
+to introduce new users to a group DM to implicitly convert it to a group room, or they
+simply may be unable to implicitly or explicity change the room type.
 
-**Opponent Users**: From the perspective of a client, the users other than the current
-user in a chat. Typically, the current user of a client will be the one which is logged
-in (and if a client supports multiple accounts, the active session within that client
-instead).
+**Room Name**: The title or human-focused textually distinguishing factor for the room. It
+may be automatically generated based on the room participants.
 
-**Chat Name**: The title or human-focused textually distinguishing factor for the chat. It
-may be automatically generated based on the chat members.
+**Room Avatar**: A picture or graphic associated with the room, usually in combination with
+a room name. Room avatars can be automatically generated based on the room participants.
 
-**Chat Avatar**: A picture or graphic associated with the chat, usually in combination with
-a chat name. Chat avatars can be automatically generated based on the chat members.
-
-**Chat Member**: A joined user in the chat. Note that this may have implications on the
+**Room Participant**: A joined user in the room. Note that this may have implications on the
 associated MLS Members in the MLS Group for the user's devices.
 
+**Roster**: Typically, a list of users actively participating in the room (who have at least
+one MLS client in the corresponding MLS group.
```

Comment:
we don't have consensus on "roster", but instead have elected to never use the word.